runtimes/alibi-explain/mlserver_alibi_explain/explainers/black_box_runtime.py

```diff
@@ -38,16 +38,15 @@ def __init__(self, settings: ModelSettings, explainer_class: Type[Explainer]):
         super().__init__(settings, explainer_settings)
 
     async def load(self) -> bool:
-        # get the metadata of the underlying inference model via v2 metadata endpoint
-        self.infer_metadata = remote_metadata(construct_metadata_url(self.infer_uri))
+        self.infer_metadata = None
```

Comment:
also given that perhaps the main fix is to make `_load_from_uri` async. I am not sure if it is still required to move this logic to `_infer_impl`...?

Reply:
Just in case there are cases or updates to Alibi where we can not call the model immediately probably best to leave the move. I will remove the setting to None.

Reply:
`_load_from_uri` is going to call `_infer_impl` and infer_metadata could come in the future from the artifact etc. i.e. outside the call to infer.

Reply:
if we change the process suggest we change it at that point